package.json

```diff
@@ -53,9 +56,9 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/capacitor-community/native-market"
+    "url": "https://github.com/djonmayer/native-market"
   },
   "bugs": {
-    "url": "https://github.com/capacitor-community/native-market/issues"
+    "url": "https://github.com/djonmayer/native-market/issues"
```

Comment:
this should not be changed. the repo is still under capacitor-community.

Reply:
I pushed a change that reverts the change in package url